esignet-ida115-integration-impl/src/test/java/io/mosip/ida115/esignet/integration/service/IdaAuthenticatorImplTest.java

```diff
@@ -128,6 +158,59 @@ public void doKycAuth_withValidDetails_thenPass() throws Exception {
 		IdaKycResponse idaKycAuthResponse = new IdaKycResponse();
 		idaKycAuthResponse.setAuthToken("authToken1234");
 		idaKycAuthResponse.setKycStatus(true);
+		idaKycAuthResponse.setIdentity("12345");
+
+		IdaResponseWrapper<IdaKycResponse> idaResponseWrapper = new IdaResponseWrapper<>();
+		idaResponseWrapper.setResponse(idaKycAuthResponse);
+		idaResponseWrapper.setTransactionID("TRAN123");
+		idaResponseWrapper.setVersion("VER1");
+
+		ResponseEntity<IdaResponseWrapper<IdaKycResponse>> responseEntity = new ResponseEntity<IdaResponseWrapper<IdaKycResponse>>(
+				idaResponseWrapper, HttpStatus.OK);
+
+		Mockito.when(restTemplate.exchange(Mockito.<RequestEntity<Void>>any(),
+				Mockito.<ParameterizedTypeReference<IdaResponseWrapper<IdaKycResponse>>>any()))
+				.thenReturn(responseEntity);
+		
+		Mockito.when(tokenIdManager.generateTokenId(Mockito.anyString(), Mockito.anyString())).thenReturn("1122334455");
+
+		KycAuthResult kycAuthResult = idaAuthenticatorImpl.doKycAuth("relyingId", "clientId", kycAuthDto);
+
+		Assert.assertEquals(kycAuthResult.getKycToken(), kycAuthResult.getKycToken());
```

Comment:
Why are we asserting the same value? @LoganathanSekar7627 
Its a positive test case, lets check for null/empty values in the kycAuthResult